CHANGELOG.md

```diff
@@ -4,23 +4,42 @@
 `psql` with the `-X` flag to prevent any `.psqlrc` commands from
 accidentally triggering the load of a previous DB version.**
 
-## Latest
+## Unreleased
+
+**Minor features**
+* #2736 Support adding columns to hypertables with compression enabled
+
+## 2.0.1 (2021-01-28)
+
+This maintenance release contains bugfixes since the 2.0.0 release.
+We deem it high priority for upgrading.
+
+In particular the fixes contained in this maintenance release address
+issues in continuous aggregates, compression, JOINs with hypertables
+and when upgrading from previous versions.
 
 **Bugfixes**
+* #2772 Always validate existing database and extension
+* #2780 Fix config enum entries for remote data fetcher
+* #2806 Add check for dropped chunk on update
+* #2828 Improve cagg watermark caching
 * #2842 Do not mark job as started when seting next_start field
 * #2845 Fix continuous aggregate privileges during upgrade
 * #2851 Fix nested loop joins that involve compressed chunks
+* #2860 Fix projection in ChunkAppend nodes
+* #2861 Remove compression stat update from update script
```

Comment:
Add https://github.com/timescale/timescaledb/pull/2874 and thanks to https://github.com/timescale/timescaledb/issues/2873